corpus/start.sh

```diff
@@ -6,6 +6,9 @@ echo "started setup"
 python manage.py makemigrations
 python manage.py migrate --no-input
 
+# Import Config data
+python manage.py loaddata config_db.json
```

Comment:
This will essentially load this every time the container starts right? Do we want to do that, or should we just run this manually whenever needed?

Only issue this could potentially have is that if the container stops due to some issue and restarts, will it create duplicate records?

Reply:
This is incorrect and needs to be changed anyway. This is run when the container is built, not when its run for the first time. For that matter, pretty much all the commands in this script don't have any use when its run during buildtime.

Making migrations, migrating the migrations and loading data all require a connection to the DB. There's no DB connected when the image is being built, so I don't think this has any reason to be here. Instead, we should put this startup script on the `docker-compose.yml` startup command/entrypoint.

Also, I'm pretty sure it won't create duplicate records because we define the primary key value as well in the data dump.

Reply:
Hmm yeah you are right. Ideally, we should remove the last line from the Dockerfile itself, so that we don't cause any errors while building the image itself.

Reply:
For now, running the prod docker compose does seem to work, so I'll approve it for now. But we do need to figure out the Dockerfile properly.